tuna/rocmlir/rocmlir_tables.py

```diff
@@ -256,6 +459,14 @@ def __init__(self, **kwargs):
         (key, value) for key, value in kwargs.items() if key in allowed_keys)
     self.set_tables()
 
+  def set_tables(self, sess_class=SessionRocMLIR):
+    """Set appropriate tables based on requirements"""
+    super().set_tables(sess_class)
+
+
+class RocMLIRDBTablesConv(RocMLIRDBTables):
```

Comment:
I was handling several operation-specific functions and tables, and I just didn't want to make a lot of run-time decisions based on the operation when I could abstract it into a class.  Turns out that now I made more init-time decisions, and I'm not sure that's an advantage.